device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttIotHubConnection.java

```diff
@@ -182,6 +183,7 @@ else if (this.config.getAuthenticationType() == DeviceClientConfig.AuthType.X509
         connectOptions.setCleanSession(SET_CLEAN_SESSION);
         connectOptions.setMqttVersion(MQTT_VERSION);
         connectOptions.setUserName(iotHubUserName);
+        connectOptions.setMaxInflight(MAX_IN_FLIGHT_COUNT);
```

Comment:
How does a customer override this default if they want to?

Reply:
Currently they can't. Its something that would could add later if a customer asks for it, but no one has asked for it yet.

Reply:
Since this is a breaking change, are we worried at all about what the impact might be on customers? Their throughput will increase. That could possibly cause problems, no?

I'm not against the change, just trying to think it through while we have the chance.